app/views/components/blockEditor.html

```diff
@@ -204,19 +204,86 @@
                 <span class="help-block help-block-hidden" ng-if="ruleValues(rule.parentBlockId).min && !ruleValues(rule.parentBlockId).max" translate>Please enter a number greater than or equal to {{ruleValues(rule.parentBlockId).min}}</span>
                 <span class="help-block help-block-hidden" ng-if="!ruleValues(rule.parentBlockId).min && ruleValues(rule.parentBlockId).max" translate>Please enter a number less than or equal to {{ruleValues(rule.parentBlockId).max}}</span>
               </div>
-              <select ng-switch-when="select" ng-model="rule.value" ng-options="v for v in ruleValues(rule.parentBlockId)" class="form-control"></select>
-              <select ng-switch-when="gender" ng-model="rule.value" class="form-control">
+              <select ng-switch-when="select" show-errors ng-model="rule.value" ng-options="v for v in ruleValues(rule.parentBlockId)" class="form-control"></select>
+              <select ng-switch-when="gender" show-errors ng-model="rule.value" class="form-control">
                 <option value="M" translate>Male</option>
                 <option value="F" translate>Female</option>
               </select>
-              <pick-a-date ng-switch-when="date" ng-model="rule.value" picker-min-date="getRangeValues(rule.parentBlockId).min" picker-max-date="getRangeValues(rule.parentBlockId).max"></pick-a-date>
+              <pick-a-date ng-switch-when="date" show-errors ng-model="rule.value" picker-min-date="getRangeValues(rule.parentBlockId).min" picker-max-date="getRangeValues(rule.parentBlockId).max"></pick-a-date>
             </div>
             <div class="clearfix visible-sm-block"></div>
             <div class="col-md-1 text-right">
               <button class="btn btn-danger" ng-click="removeRule(rule.id)" title="Delete Rule"><i class="fa fa-trash-o"></i></button>
             </div>
           </div>
         </div>
+
+        <div ng-if="block.type === 'checkboxQuestion' && disableForceSelectionRule() === false">
+          <p class="text-muted"><i class="fa fa-info-circle"></i>
+            <translate>This Rules section allows for conditional enforcement of any answer marked 'Force Selection'.</translate>
+            <translate>Rules can depend on questions of</translate> <a href="" tooltip="{{'Multiple Choice, Dropdown, Number, Date, Gender or Year in School' | translate}}"
+              translate>these types</a>
+            <translate>that are above this question.</translate>
+          </p>
+          <div class="form-horizontal">
+            <div class="form-group">
+              <div class="col-md-2">
+                <button class="btn btn-success" ng-click="addForceRule()"><i class="fa fa-plus"></i> <translate>Add Rule</translate></button>
+              </div>
+              <div class="col-md-8 rule-message-alignment">
+                <span class="pull-left" translate>Show question if:</span>
```

Comment:
@OzzieOrca Aren't the existing information headers enough?

`Rules are a great way to hide questions based on the registrant's answers to other questions. All rules created must be met before the question is displayed. Rules can depend on questions of these types that are above this question.` for normal visibility rules

and

`This Rules section allows for conditional enforcement of any answer marked 'Force Selection'. Rules can depend on questions of these types that are above this question.` for force selection rules?